pkgs/by-name/sq/sqruff/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "sqruff";
+  version = "0.11.0";
+  src = fetchurl {
+    url = "https://github.com/quarylabs/sqruff/releases/download/v{finalAttrs.version}/sqruff-linux-x86_64-musl.tar.gz";
```

Apologies, @cafkafk, I missed the missing `$` the first time around:
```suggestion
    url = "https://github.com/quarylabs/sqruff/releases/download/v${finalAttrs.version}/sqruff-linux-x86_64-musl.tar.gz";
```

---

Addressed in incoming fixes